src/jquery-sticky-header-footer.js

```diff
@@ -225,12 +224,6 @@
                 ];
 
             /**
-                Decorate original header and footer elements to
-                differentiate them from clones at runtime.
-             */
-            $(this[element]).addClass(originalClassName);
```

Comment:
The added classes weren't having the desired effect since we were adding them before the element was cloned.